ceaos/sensors/temperature_sensor.py

```diff
@@ -0,0 +1,39 @@
+"""
+This file contains a class for a temperature sensor
+"""
+from .sensor_definition import Sensor  
+from ceaos.loggers.InfluxDB import InfluxDBConnection, InfluxDBLogger
+
+
+class Temperature(Sensor):
+
+    datatype = "General_Temp"  # static variable for Temperature class
+
+    def __init__(self, temp_value=None, datatype=datatype) -> None:
```

Comment:
This is not the best way to do this. Use a default initialization for the variable in the method signature!

Reply:
Oooh, ok yup that makes more sense!